client/src/Pages/Login/LoginView.js

```diff
@@ -24,10 +24,19 @@ const LoginView = props => {
   const enteredInvalidPassword = data && data.login && !data.login.successful;
 
   if (!loading && data && data.login && data.login.successful) {
+    if(data.login.authToken != null && data.login.userId != null){
+      document.cookie = `user_session=${data.login.authToken}`
+      document.cookie = `user_Id=${data.login.userId}`
+    }
+    // using regex to get the remove the strings and only getting the value that is stored
+    const user_session = document.cookie.replace(/(?:(?:^|.*;\s*)user_session\s*=\s*([^;]*).*$)|^.*$/, "$1");
```

Comment:
rule of three suggests that you abstract this functionality into a helper method. https://blog.adrianbolboaca.ro/2015/02/refactoring-rule-of-three/

Reply:
I'll have a look into helper methods and try add them in will also try fix the issue with me writing inconsistently.